packages/server/src/automations/utils.ts

```diff
@@ -40,39 +40,35 @@ function loggingArgs(job: AutomationJob) {
 }
 
 export async function processEvent(job: AutomationJob) {
-  return tracer.trace(
-    "processEvent",
-    { resource: "automation" },
-    async span => {
-      const appId = job.data.event.appId!
-      const automationId = job.data.automation._id!
-
-      span?.addTags({
-        appId,
-        automationId,
-        job: {
-          id: job.id,
-          name: job.name,
-          attemptsMade: job.attemptsMade,
-          opts: {
-            attempts: job.opts.attempts,
-            priority: job.opts.priority,
-            delay: job.opts.delay,
-            repeat: job.opts.repeat,
-            backoff: job.opts.backoff,
-            lifo: job.opts.lifo,
-            timeout: job.opts.timeout,
-            jobId: job.opts.jobId,
-            removeOnComplete: job.opts.removeOnComplete,
-            removeOnFail: job.opts.removeOnFail,
-            stackTraceLimit: job.opts.stackTraceLimit,
-            preventParsingData: job.opts.preventParsingData,
-          },
-        },
-      })
+  return tracer.trace("processEvent", async span => {
+    const appId = job.data.event.appId!
+    const automationId = job.data.automation._id!
+
+    span.addTags({
+      appId,
+      automationId,
+      job: {
+        id: job.id,
+        name: job.name,
+        attemptsMade: job.attemptsMade,
+        attempts: job.opts.attempts,
+        priority: job.opts.priority,
+        delay: job.opts.delay,
+        repeat: job.opts.repeat,
+        backoff: job.opts.backoff,
+        lifo: job.opts.lifo,
+        timeout: job.opts.timeout,
+        jobId: job.opts.jobId,
+        removeOnComplete: job.opts.removeOnComplete,
+        removeOnFail: job.opts.removeOnFail,
+        stackTraceLimit: job.opts.stackTraceLimit,
+        preventParsingData: job.opts.preventParsingData,
```

Comment:
These have been unnested because they appear to get ignored when they're nested past a certain level.